src/porepy/fracs/split_grid.py

```diff
@@ -80,36 +80,36 @@ def split_fractures(
 
         # We split all the faces that are connected to a lower-dim grid. The new
         # faces will share the same nodes and properties (normals, etc.)
-        face_cells_modified = split_faces(gh, matrix_list)
+        face_cells_modified = split_faces(sd_primary, matrix_list)
 
-        for gl, matrix in zip(low_dim_neigh, face_cells_modified):
-            sd_pairs[(gh, gl)] = matrix
+        for sd_secondary, matrix in zip(low_dim_neigh, face_cells_modified):
+            sd_pairs[(sd_primary, sd_secondary)] = matrix
 
         # We now find which lower-dim nodes correspond to which higher- dim nodes. We
         # split these nodes according to the topology of the connected higher-dim
         # cells. At a X-intersection we split the node into four, while at the
         # fracture boundary it is not split.
 
-        gl_2_gh_nodes = []
-        for g in low_dim_neigh:
+        secondary_2_primary_nodes = []
```

Comment:
I personally don't like the use of "2" for "to".

Reply:
Agree. Changed..